Jenkinsfile

```diff
@@ -59,6 +59,21 @@ pipeline {
                  }
 		    }
 		}
+		stage('DeployProdStack') {
+	        when {
+                buildingTag();
+	        }
+		    steps {
+	            script {
+                    withKubeConfig([credentialsId: "prod-kube-config"]) {
+                        sh('helm repo update && helm dependency update helm-chart/banzai-nres/ '+
+                                '&& helm upgrade --install banzai-nres helm-chart/banzai-nres ' +
+                                '--set image.tag="${GIT_DESCRIPTION}" --values=helm-chart/banzai-nres/values-prod.yaml ' +
+                                '--force --wait --timeout=3600')
```

Comment:
I like to add the `--atomic` flag as well. And you can probably remove the `--wait` flag. Also would recommend explicitly setting the `--namespace=prod` flag as well.

```
      --atomic                       if set, upgrade process rolls back changes made in case of failed upgrade. The --wait flag will be set automatically if --atomic is used
```